google/cloud/aiplatform/models.py

```diff
@@ -2068,6 +2691,15 @@ def deploy(
             explanation_parameters,
         )
 
+        if endpoint and endpoint.__class__ == "PrivateEndpoint"():
```

Comment:
I've never encountered this syntax, how does this string literal invocation work: `"PrivateEndpoint"()`

Preference for `isinstance(endpoint, PrivateEndpoint)`

Reply:
Fixed